src/main/res/values/themes.xml

```diff
@@ -19,6 +19,8 @@
         <item name="drawerArrowStyle">@style/Widget.Syncthing.DrawerArrowToggle</item>
 
         <item name="android:actionBarSize">@dimen/abc_action_bar_default_height_material</item>
+
+        <item name="android:spinnerStyle">@android:style/Widget.Holo.Light.Spinner</item>
```

Comment:
Whats the reason you're setting Holo spinner here? It should use the material spinner by default, which es exactly what we want.